bob/src/api/mod.rs

```diff
@@ -145,12 +145,25 @@ pub(crate) struct NodeConfiguration {
 }
 
 #[derive(Debug, Serialize)]
-pub(crate) struct SpaceInfo {
+struct Space {
     total_disk_space_bytes: u64,
     free_disk_space_bytes: u64,
     used_disk_space_bytes: u64,
+}
+
+#[derive(Debug, Serialize)]
+pub(crate) struct SpaceInfo {
```

Comment:
Please update openapi in `config-examples`

Reply:
I think it would be good to integrate [utoipa](https://github.com/juhaku/utoipa) for openapi auto-gen at some point